src/utils/utils.ts

```diff
@@ -352,7 +371,7 @@ export const ACITriggerSettings = <
         return data;
       }
 
-      return queryMembers(channel, query, (data) => {
+      return queryMembersDebounced(channel, query, (data) => {
```

Comment:
@nhannah @virdesai Can I get some help here for typescript generics? Not able to resolve some errors here